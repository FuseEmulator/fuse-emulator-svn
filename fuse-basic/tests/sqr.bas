10 PRINT SQR 0
20 PRINT SQR 1
30 PRINT SQR 4
40 PRINT ( SQR 9.85 ) ^ 2
50 PRINT SQR 1.44 * SQR 1.44
60 PRINT EXP( 0.5 * LN 2.53 ) - SQR 2.53




