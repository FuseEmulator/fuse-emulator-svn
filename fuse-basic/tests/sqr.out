0
1
2
9.85
1.44
0
0 OK
